tests/test_atomic_requests.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import unicode_literals
+
+from django.db import connection, connections, transaction
+from django.test import TestCase
+from django.utils.unittest import skipUnless
+from rest_framework import status
+from rest_framework.exceptions import APIException
+from rest_framework.response import Response
+from rest_framework.test import APIRequestFactory
+from rest_framework.views import APIView
+from tests.models import BasicModel
+
+
+factory = APIRequestFactory()
+
+
+class BasicView(APIView):
+    def get(self, request, *args, **kwargs):
```

Comment:
It might make the tests slightly more obvious if we use `post` for this state changing operation.